pkg/types/openstack/machinepool.go

```diff
@@ -95,56 +85,3 @@ type RootVolume struct {
 	// +optional
 	Zones []string `json:"zones,omitempty"`
 }
-
-// -=**
-// FailureDomain and its types are part of a Tech Preview feature, and may change before they're moved to openshift/api
-// **=-
-
-// FailureDomain defines a set of correlated fault domains across different
-// OpenStack services: compute, storage, and network.
-type FailureDomain struct {
-	// ComputeAvailabilityZone is the name of a valid nova availability zone. The server will be created in this availability zone.
-	// +optional
-	ComputeAvailabilityZone string `json:"computeAvailabilityZone"`
-
-	// StorageAvailabilityZone is the name of a valid cinder availability
-	// zone. This will be the availability zone of the root volume if one is
-	// specified.
-	// +optional
-	StorageAvailabilityZone string `json:"storageAvailabilityZone"`
-
-	// Ports defines a set of port targets which can be referenced by machines in this failure domain.
-	//
-	// +optional
-	PortTargets []NamedPortTarget `json:"portTargets"`
-}
-
-// NamedPortTarget includes an ID to characterize a PortTarget with its
-// intended purpose. If ID is set to "control-plane", then the PortTarget will
-// replace the default cluster primary network (or the machinesSubnet if
-// defined) as the first network for the machine.
-type NamedPortTarget struct {
-	ID         string `json:"id"`
-	PortTarget `json:",inline"`
-}
-
-// PortTarget defines, directly or indirectly, one or more subnets where to attach a port.
-type PortTarget struct {
```

Comment:
PortTarget is being used for the dual-stack support:
https://github.com/openshift/installer/pull/6797/files#diff-0b0e430b5843d7e62d8138813bc0517be9584d7c77e3ad30686b3fc3dde02e21L132-L138

Reply:
Or even better if we can merge your PR first